bin/README.md

```diff
@@ -0,0 +1,68 @@
+# Mini Tree Server
+
+Mini tree server is wrapped 
```

Comment:
What does it mean for it to be wrapped?

Reply:
I think it's cropped. I will add more description